2697. Lexicographically Smallest Palindrome/solution.py

```diff
@@ -0,0 +1,33 @@
+class Solution:
+    # Takes in the string s and turns it into a palindrome
+    # by replacing as few letters as possible. Breaks tie by
+    # checking the lexicographical value of the current letter.
+    # Returns the palindrome string.
+    def makeSmallestPalindrome(self, s: str) -> str:
+        # turn the string into a list for easier manipulation. This allows
```

Comment:
Better to say that turning an immutable string into a list allows you to modify the letters.